File: C Language/Assignments/Assi 3/fact.c
//Find factorial of number. 
#include<stdio.h>
void main(){
    int n;
    int fact=1;
    printf("The the number");
    scanf("%d",&n);       
    for(int i=1;i<=n;i++){
        fact=fact*i;
    }
    printf("the factorial of number is %d",fact);
}

File: Java work/Code Practice/fact.java
package Other;
import java.util.Scanner;

public class fact {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int p = sc.nextInt();
        int facter = 1;
        for (int i = 1; i <= p; i++) {

            facter = facter * i;

        }
        System.out.println("factorial of the number" + facter);
    }

}
